src/NzbDrone.Core/Applications/LazyLibrarian/LazyLibrarian.cs

```diff
@@ -78,6 +78,14 @@ public override void AddIndexer(IndexerDefinition indexer)
             var lazyLibrarianIndexer = BuildLazyLibrarianIndexer(indexer, indexer.Protocol);
 
             var remoteIndexer = _lazyLibrarianV1Proxy.AddIndexer(lazyLibrarianIndexer, Settings);
+
+            if (remoteIndexer == null)
+            {
+                _logger.Warn("Remote indexer not found, failed to add {0} [{1}].", indexer.Name, indexer.Id);
```

Comment:
Should we also specify which app it was not found for?

Reply:
Yes but multiple apps. So probably should log the prowlarr app name defined too just so it's clear and easier to troubleshoot